bbq/vm/vm.go

```diff
@@ -1652,6 +1723,12 @@ func (vm *VM) run() {
 			opStatement(vm)
 		case opcode.InstructionTemplateString:
 			opStringTemplate(vm, ins)
+		case opcode.InstructionGetTypeIndex:
+			opGetTypeKey(vm, ins)
+		case opcode.InstructionSetTypeIndex:
+			opSetTypeKey(vm, ins)
+		case opcode.InstructionRemoveTypeIndex:
+			opRemoveTypeKey(vm, ins)
```

Comment:
Nit: maybe rename the function names to also use the term "Index", instead of "Key", to be consistent with the instruction names.